### PR TITLE
Add test_bastion_javascript

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/pr_checkout.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+
+cd plugin/
```
Comment (domcleal) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/pr_checkout.sh]: This shouldn't be in a generic "pr_checkout", perhaps "plugin_pr_checkout" or use the root directory.

Comment (ehelms) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/pr_checkout.sh]: Tried a multi-purpose solution for plugin and non-plugin pr checkouts -- let me know what you think. If you would still prefer they be separated logic for plugins and non-plugins I'll generate two scripts.

Comment (domcleal) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/pr_checkout.sh]: I think separate's probably going to be best, or it needs to take a variable.. it's probably best not to assume a certain layout, or what type of PR we're testing.  (Perhaps it's testing a non-plugin PR against the plugin.)

**Comment by ehelms** [comment]
Updated

**Comment by domcleal** [comment]
Merged, thanks @ehelms.

**Pull request merged**